.circleci/config.yml

```diff
@@ -154,8 +157,8 @@ commands:
             wget https://raw.githubusercontent.com/algorand/go-algorand/rel/stable/cmd/updater/update.sh && chmod 744 update.sh
             ./update.sh -i -c nightly -n -d ./ -p /usr/local/go/bin
             export GOPATH=/usr/local/go/
-      - run: make e2e-nightly
-      - run: make e2e-filter-test-nightly
+      - run: make e2e
+      - run: make e2e-filter-test
```

Comment:
Change these back?
```suggestion
      - run: make e2e-nightly
      - run: make e2e-filter-test-nightly
```

Reply:
since we set the channel this script and picked up in docker-compose file. I didn't think we need e2e-nightly or e2e-filter-test-nightly anymore.

Reply:
Got it.